app/Menu.js

```diff
@@ -1,157 +1,182 @@
-	  label: 'Save Settings as...'
-	}));
-	file_submenu.append(new gui.MenuItem({ 
-	  label: 'Exit',
-	  tooltip:"Closes all the windows and exits the program safely",
-	  click: function(){
-	    Logger.debug('Closing all windows!');
-	    gui.App.closeAllWindows();
-	  }
-	}));
-
-	// Append file menu as a Submenu
-	main_menu.append(
-	    new gui.MenuItem({
-	        label: 'File',
-	        submenu: file_submenu 
-	    })
-	);
-
-	// ===Window Submenu ===
-	var window_submenu=new gui.Menu();
-
-	window_submenu.append(new gui.MenuItem({
-	  label:'Console',
-	  type:'checkbox'
-	}));
-
-	window_submenu.append(new gui.MenuItem({
-	  label:'3D View',
-	  type:'checkbox',
-    click: function(){
-      gui.Window.open('./windows/3dView.html',{
-       focus: true,
-       position: 'center',
-       width: 900,
-       height: 500,
-       toolbar: false
-      });
-      Logger.debug('Opening 3D View window');
-    },
-    modifiers: 'ctrl-shift',
-    key: 'v'
-	}));
-
-	window_submenu.append(new gui.MenuItem({
-	  label:'Autonomous Level',
-	  type:'checkbox'
-	}));
-	window_submenu.append(new gui.MenuItem({
-	  label:'Gains Adjust',
-	  type:'checkbox',
-    click: function(){
-      gui.Window.open('./windows/gainsAdjust.html',{
-       focus: true,
-       position: 'center',
-       width: 900,
-       height: 400,
-       toolbar: false
-      });
-      Logger.debug('Opening gains adjust window');
-    },
-    modifiers: 'ctrl-shift',
-    key: 'g'
-	}));
-	window_submenu.append(new gui.MenuItem({
-	  label:'Data Entry',
-	  type:'checkbox'
-	}));
-	window_submenu.append(new gui.MenuItem({
-	  label:'Altitude Graph Entry',
-	  type:'checkbox'
-	}));
-
-	// Append window menu as a Submenu
-	main_menu.append(
-	    new gui.MenuItem({
-	        label: 'Window',
-	        submenu: window_submenu
-	    })
-	);
-
-	// ===Settings Submenu ===
-	var settings_submenu=new gui.Menu();
-
-	settings_submenu.append(new gui.MenuItem({
-	  label:'Network Settings'
-	}));
-	settings_submenu.append(new gui.MenuItem({
-	  label:'App Settings'
-	}));
-	
-	// Append settings menu as a Submenu
-	main_menu.append(
-	    new gui.MenuItem({
-	        label: 'Settings',
-	        submenu: settings_submenu
-	    })
-	);
-
-	// ===Help Submenu ===
-	var help_submenu=new gui.Menu();
-
-	help_submenu.append(new gui.MenuItem({
-	  label:'About',
-	  click:function(){
-
-	  }
-	}));
-
-	help_submenu.append(new gui.MenuItem({
-	  label:'Documentation',
-	  click:function(){
-	  	// Create a new window and get it
-		var documentation_window = gui.Window.open('http://www.uwarg.com/projects/groundstation/',{
-			focus: true
-		});
-		Logger.debug('Opening documentation window');
-	  },
-	  key:'F1'
-	}));
-	
-	// Append settings menu as a Submenu
-	main_menu.append(
-	    new gui.MenuItem({
-	        label: 'Help',
-	        submenu: help_submenu
-	    })
-	);
-
-	return main_menu;
+var Logger = require('./util/Logger');
+
+module.exports = function (gui) {
```

Comment:
Try not to make indentation and space changes in the same PR as a new feature since it makes figuring out what you did harder.